rep-2000.rst

```diff
@@ -575,8 +695,8 @@ References and Footnotes
    (https://github.com/ros2/ros2/blob/dashing/ros2.repos)
 .. [6] Eloquent ROS 2 Repos File
    (https://github.com/ros2/ros2/blob/master/ros2.repos)
-.. [7] Connext DDS 6.0.0 Migration
-   (https://github.com/ros2/rmw_connext/pull/351)
+.. [7] Connext DDS 6.0.0 Support
+   (https://github.com/ros2/rmw_connext/issues/375)
```

Comment:
This can just be removed from the REP, since it doesn't look like it is going to happen.

Reply:
This footnote is still referenced by older distros in this document. If we remove this, I suggest we do it in a separate PR.